packages/entities/src/lib/update.mutation.ts

```diff
@@ -243,3 +243,85 @@ export function upsertEntities<
     };
   };
 }
+
+/**
+ * Update entities ids
+ *
```

Comment:
Update the docs, please

Reply:
sure thing. I wanted to make sure the code looks good before starting with the docs.